### ISSUE TITLE
Trust anchor for certification path not found

### ISSUE THREAD

**Issue by prankousky** [thrillfall/nextcloud-gpodder#82]
Hi,

I am trying to link my `AntennaPod` to `nextcloud-gpodder`. When I start AntennaPod, then go to 

Settings > Synchronize > Chose Provider for Sync (**or similar**; my language is set to German

I get to log in. 

For `Hostname`, I put `https://<my-ip>:2443` (2443 being the port `docker` exposes to the nextcloud container's port 443). When I then click `Login`, I get 

```
java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
```

I thought, perhaps this was related to the fact that I use `nginx-reverse-proxy` to use a local domain for `https://<my-ip>:2443`, so that I can just access nextcloud through `https://nextcloud.internal` (without any port) and it will forward any requests to that IP:port. This internal domain has a self-signed certificate. However, this certificate is **only** used when accessing it through the reverse proxy; when I open `https://<ip>:2443` in my browser, it uses whatever the nextcloud docker container uses by default as the certificate.

* NextCloud Version 23.0.2
* Gpodder Sync Version 3.4.0
* AntennaPod Version 2.6.1 (50eee8848)

I now updated to NextCloud Version 23.0.6, which seems to  not make  any difference.

Do I need to change / add any configuration and / or use a different URL? I tried `https://<ip>:2443/index.php/apps/gpoddersync` as well, no difference.

Can you please help me fix this? Thank you in advance for your support :)

**Comment by prankousky** [comment]
Thank you for your help @JonOfUs 

Unfortunately, my docker image (`linuxserver/nextcloud`) does not let me connect without https - or I am just not doing it correctly.

I will try and set up another image for testing this. Perhaps even switch to the official image, if things work better that way. This might take some time, but I'll report back if I got results :)

**Comment by prankousky** [comment]
UPDATE: I managed to try the `nextcloud` appimage, which uses http (not https) by default. And it worked fine. I can grant access to this nextcloud instance to `AntennaPod` and I believe I can sync.

~~*I believe*, because I cannot find any files that might belong to `nextcloud-gpodder`. It is essential to me that I can backup this data.
There is `/data/custom_apps/gpoddersync`, which includes multiple folders; however, all these folders contain only `.php` files. The `Db` folder is empty (even though `AntennaPod` states that I synced successfully.~~

I realize that this data is backed up via MYSQL. So I assume the issue is closed. Thank you!
